lib/index.js

```diff
@@ -135,10 +135,16 @@ function globCodeSamples(dir) {
 }
 
 function globObject(dir) {
-  var files = glob.sync(dir + '*.yaml');
+  var files = glob.sync(dir + '**/*.yaml');
   var object = {};
   _.each(files, function (file) {
-    object[path.parse(file).name] = readYaml(file);
+    var name = path.parse(file).name;
+    
+    if ($name in obj) {
+      throw new Error($name + " definition already exists");
```

Comment:
@slavcodev is it php? 😄 
the same in the next line (`$` sign before variable name)